test/functional/Java21Only/src/org/openj9/test/jep442/upcall/UpcallMHWithMixedSigStruTests.java

```diff
@@ -21,25 +21,30 @@
  */
 package org.openj9.test.jep442.upcall;
 
-import org.testng.annotations.Test;
 import org.testng.Assert;
-import org.testng.AssertJUnit;
-
-import java.lang.invoke.MethodHandle;
-import java.lang.invoke.VarHandle;
+import org.testng.annotations.Test;
 
 import java.lang.foreign.Arena;
-import java.lang.foreign.Linker;
 import java.lang.foreign.FunctionDescriptor;
+import java.lang.foreign.Linker;
 import java.lang.foreign.GroupLayout;
```

Comment:
These two imports should be reversed; `ValueLayout.JAVA_BOOLEAN` is unused.

Reply:
Fixed.